CLAUDE.md

```
@@ -14,6 +14,19 @@ yarn format         # prettier, printWidth 160
 yarn format:check   # what CI runs
 ```
 
+Before a release, against a real apps checkout (NOT in CI — `../apps` is a private working
+checkout, so a job depending on it would be red for reasons nobody here could act on):
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)